modules/RTC/RTCUtils.js

```diff
@@ -821,16 +821,13 @@ var RTCUtils = {
                             : SDPUtil.filter_special_chars(id));
                 };
                 // DTLS should now be enabled by default but..
-                this.pc_constraints = {'optional': [
-                    {'DtlsSrtpKeyAgreement': 'true'}
-                ]};
+                this.pc_constraints = {'optional': [] };
+
                 if (options.useIPv6) {
                     // https://code.google.com/p/webrtc/issues/detail?id=2828
                     this.pc_constraints.optional.push({googIPv6: true});
                 }
-                if (RTCBrowserType.isAndroid()) {
```

Comment:
Not in recent versions. I don't know when it was last required to use this constraint, but it wasn't recently.